src/features/security-cloud-game/SecurityCloudGame.scss

```diff
@@ -49,6 +53,10 @@
     left: -9999px;
 }
 
+.bdl-SecurityCloudGame-tooltip.bdl-Tooltip {
```

Comment:
is the `.bdl-Tooltip` part needed? or could it be removed to lower specificity?

Reply:
Yes, it's needed. I tried removing `.bdl-Tooltip` but it didn't work.